files/SZWESAMP/ZWEIMVSP

```diff
@@ -0,0 +1,43 @@
+//ZWEIMVSP JOB {zowe.environments.jclHeader}
+//*
+//* This program and the accompanying materials are made available
+//* under the terms of the Eclipse Public License v2.0 which
+//* accompanies this distribution, and is available at
+//* https://www.eclipse.org/legal/epl-v20.html
+//*
+//* SPDX-License-Identifier: EPL-2.0
+//*
+//* Copyright Contributors to the Zowe Project. 2020, 2020
+//*
+//*********************************************************************
+//*
+//* This job is used to create datasets used by a Zowe "instance"
+//* Instances represent a configuration of Zowe, different from the
+//* "runtime" datasets that are created upon install of Zowe / SMPE.
+//*
+//* Consider if the Zowe Cross Memory Server
+//*  - is used: run this and consider to run ZWEIMVSZ
+//*  - not used: you can skip this JCL
+//*
+//* If your chosen value of 'zowe.setup.dataset.authLoadlib' is not
+//*  equal to 'zowe.setup.dataset.prefix' + 'SZWEAUTH',
+//*  then you must also run ZWEIMVSA.
+//*
+//*********************************************************************
+//MKPDSE EXEC PGM=IKJEFT01
+//SYSTSPRT DD SYSOUT=A
+//SYSTSIN DD *
+ALLOC NEW DA('{zowe.setup.dataset.parmlib}') +
+dsntype(library) dsorg(po) recfm(f b) lrecl(80) +
+unit(sysallda) space(15,15) tracks
+/*
+//MEMBCPY EXEC PGM=IDCAMS
+//  SET OUTPARM={zowe.setup.dataset.parmlibMembers.zis}
+//SYSPRINT DD  SYSOUT=*
+//PARMIN  DD DISP=SHR,
+//  DSN={zowe.setup.dataset.prefix}.SZWESAMP(ZWESIP00)
+//PARMOUT DD DISP=OLD,
+//  DSN={zowe.setup.dataset.parmlib}(&OUTPARM)
+//SYSIN DD *
+  REPRO INFILE(PARMIN) OUTFILE(PARMOUT)
```

Comment:
@1000TurquoisePogs Would this help for `zwe init stc`? `ZWEMCOPY` is using repro and I think (but I am not sure), IDCAMS is after all using TSO commands anyway. If `ZWEMCOPY` is better solution, then the "IDCAMS JCL" might be also better solution.